secant/Features/BackupFlow/Views/ValidationFailedView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ValidationFailed.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 12/22/21.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct ValidationFailedView: View {
+    var store: RecoveryPhraseValidationStore
+
+    var body: some View {
+        WithViewStore(store) { viewStore in
+            GeometryReader { proxy in
+                VStack {
+                    VStack(alignment: .center, spacing: 20) {
+                        Text("Ouch, sorry, no.")
+                            .font(.custom(FontFamily.Rubik.regular.name, size: 30))
+                            .fixedSize(horizontal: false, vertical: true)
+                    }
+                    .padding(.bottom, 20)
+
+                    CircularFrame()
+                        .backgroundImage(
+                            Asset.Assets.Backgrounds.callout1.image
+                        )
+                        .frame(
+                            width: proxy.size.width * 0.84,
+                            height: proxy.size.width * 0.84
+                        )
+                        .badgeIcon(.error)
+
+                    Spacer()
+                    VStack(alignment: .center, spacing: 40) {
+                        VStack(alignment: .center, spacing: 20) {
+                            Text("Your placed words did not match your secret recovery phrase.")
+                                .bodyText()
+                                .fixedSize(horizontal: false, vertical: true)
+
+                            Text("Remember, you can't recover your funds if you lose (or incorrectly save) these 24 words.")
+                                .bodyText()
+                                .fixedSize(horizontal: false, vertical: true)
+                        }
+
+                        Button(
+                            action: { viewStore.send(.reset) },
+                            label: { Text("I'm ready to try again") }
+                        )
+                            .activeButtonStyle
+                            .frame(height: 60)
```

Comment:
When a view initializer is multi line, making the trailing paren on a new line, the convention we've been following is to keep the modifiers indented to that same level so they aren't "dangling".

```suggestion
                        Button(
                            action: { viewStore.send(.reset) },
                            label: { Text("I'm ready to try again") }
                        )
                        .activeButtonStyle
                        .frame(height: 60)
```